compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -45,6 +45,21 @@ def BufferizeCopyOnlyDispatchesPass :
   }];
 }
 
+def BufferizeDispatchTensorLoadStorePass :
+  InterfacePass<"iree-codegen-bufferize-dispatch-tensor-load-store", "mlir::FunctionOpInterface"> {
+  let summary =
+      "Bufferize the flow.dispatch.tensor.load/store ops at dispatch boundaries";
+  let description = [{
+    Pass to bufferize the edges of dispatch regions, converting
+    flow.dispatch.tensor.load ops to iree_codegen.load_from_memref, and
+    flow.dispatch.tensor.store ops to iree_codegen.store_to_memref.
+  }];
+  let dependentDialects = [
+    "IREE::Codegen::IREECodegenDialect",
+    "memref::MemRefDialect"
+  ];
+}
```

Comment:
nit: replace `flow.` with `iree_tensor_ext.`. They are in TensorExt dialect now.